README.md

```diff
@@ -149,13 +165,19 @@ Either a single service or multiple services can be provided as an array.
 
 The name of the service the command should be run within. If the docker-compose command would usually be `docker-compose run app test.sh` then the value would be `app`.
 
+### `push`
+
+A list of services to push in the format `service:image:tag`. If an image has been pre-built with the build step, that image will be re-tagged, otherwise docker-compose's built in push operation will be used. 
+
+Be aware that there is a race condition on tagging prebuilt images and pushing them if multiple push steps run in parallel. It's advisable to use a concurrency group in this situation.
```

Comment:
This sounds scary! Is there anything specific about this plugin that introduces a race condition? Or is that a general "don't push two things at once"?